src/github.com/matrix-org/dendrite/roomserver/input/state.go

```diff
@@ -10,7 +10,7 @@ import (
 // calculateAndStoreState calculates a snapshot of the state of a room before an event.
 // Stores the snapshot of the state in the database.
 // Returns a numeric ID for that snapshot.
```

Comment:
- `s/calculateAndStoreState/calculateAndStoreStateBeforeEvent/`
- When you say "returns a numeric ID for that snapshot" that's unhelpful, I can see it's a snapshot NID. More importantly, this snapshot is the state of the room **after** the given event? Before?